src/pages/newblog/NewBlog.css

```diff
@@ -0,0 +1,32 @@
+.errorMsg {
+    max-width: 350px;
+    color: #f21e08;
+    padding: 2px 0;
+    margin-top: 2px;
+    font-size: 14px;
+    font-weight: 500;
+}
+
+.form-control {
```

Comment:
Je kunt je bouwblokken binnen het formulier beter altijd een width: 100% geven, zodat ze altijd zoveel mogelijk ruimte innemen, maar het **formulier** te laten bepalen wat de kaders zijn, want dat is per pagina anders. Zo behoud je de herbruikbaarheid van je input-elementen :)